lib/data/table.js

```diff
@@ -19,7 +19,19 @@ class DataTable {
       tempObj.toString = () => JSON.stringify(tempObj);
       arrayCounter++;
     });
-    this.rows.push(tempObj);
+    this.rows.push({ skip: false, data: tempObj});
+  }
+
+  xadd(array) {
```

Comment:
where is xadd used?
probably it should be documented, right?

Reply:
Updated.